src/actions/ui.js

```diff
@@ -76,6 +80,19 @@ export function uiContributionGetData(token, entity) {
   return getPrDataAction(token, entity)
 }
 
+/**
+ * Get the curation in the given PR relative to the specified coordinates
+ * @param  {} token
```

Comment:
I'm sure that I added feedback on comments like these somewhere but don't see them. The empty params are a liability. Here for example, the comment param list does not match the actual param list. The comment params don't add any value. Unless the naming of things is deficient, the code should be self describing. 

The challenges with adding comments 
* easy to get out of date
* imply that things are more complicated than they otherwise seem from the code naming etc

Comments are great if their value outweighs the above issues. If we were doing APIs and publishing API doc then it would IMHO be a different situation. Even then, ideal is that the method and arg naming is semantically useful.